src/json/program/AfterUserEventProcessedTriggerContext.schema.ts

```diff
@@ -0,0 +1,30 @@
+import { JSONSchema6 } from "json-schema";
+import {
+  contextTimeField,
+  userContext,
+  customFieldsAndEventsDefinitions
+} from "./ProgramTriggerVariableContext.schema";
+
+const afterUserEventProcessedTriggerContextSchema: JSONSchema6 = {
+  $schema: "http://json-schema.org/draft-06/schema#",
+  type: "object",
+  title: "After User Event Processed Trigger Context",
+  description:
+    "Defines the trigger variables available to an AFTER_USER_EVENT_PROCESSED trigger",
```

Comment:
Make this more humanized / doc friendly

```suggestion
    "These variables are available for designing conversion rules for Event triggers",
```